data/tooling-data.yaml

```diff
@@ -1497,6 +1497,22 @@
   bowtie:
     identifier: 'cpp-jsontoolkit'
 
+- name: cypress-ajv-schema-validator
+  description: 'Cypress plugin that validates API responses against Plain JSON schemas, and whole Swagger and OpenAPI documents using the Ajv JSON Schema validator. It provides a user-friendly view of validated data, highlighting each validation error and the exact reason for the mismatch. Seamless integration, fast and lightweight.'
+  toolingTypes: ['validator']
```

Comment:
I think this tool fits better in the Testing category.

Reply:
Hey! @benjagm Thanks for the feedback! Do you mean it should be `toolingTypes: ['util-testing]` ?
Because in the original issue (#940 ) it was marked as a validator. 
![image](https://github.com/user-attachments/assets/5ffe4a8a-7b57-4102-b6f9-22d6f0868753)

Reply:
Yessir! Updating the branch with only the second comment then

Reply:
Yes! That is possible. @iamAyushChamoli can you add util-testing to the possible values?

Reply:
@benjagm @sebastianclavijo Added it to both util-testing and validator

Reply:
Thanks @iamAyushChamoli and @benjagm !

Reply:
@iamAyushChamoli , it seems that the PR is blocked because the requested change has not been marked as resolved yet.